integration_test_lite.py

```diff
@@ -131,19 +147,19 @@ def basic_request(self, url, params, needs_access_token, method='get'):
         results = await asyncio.gather(
             *[self.bad_response(endpoint) for endpoint in endpoints]
         )
-        print()
         # Return all results that weren't None
         return [result for result in results if result]
 
 
 async def main():
-    async with IntegrationTestLite() as integration_test_lite:
-        await integration_test_lite.set_access_token()
-        bad_apis = await integration_test_lite.get_bad_apis()
-
-        if bad_apis:
-            print('The following API(s) returned errors:')
-            pretty_print(bad_apis)
-            sys.exit(1)
+    integration_test_lite = IntegrationTestLite()
+    await integration_test_lite.set_access_token()
+    bad_apis = await integration_test_lite.get_bad_apis()
+
+    if bad_apis:
+        print()
+        print('The following API(s) returned errors:')
```

Comment:
Not really important, but these two lines can be combined as: `print('\nThe following API(s) returned errors:')`